catalog/app/components/Preview/Preview.js

```diff
@@ -15,10 +15,12 @@ const fallback = {
 };
 
 const loaderChain = [
+  loaders.Csv,
   loaders.Json,
   loaders.Markdown,
   loaders.Notebook,
   loaders.Parquet,
+  loaders.Vcf,
```

Comment:
OK for separate PR, but we now support XLS and XLSX as well - works like .parquet (minus the info metadata)

Reply:
ok